### ISSUE TITLE
Provide tags on hub.docker.com with major version

### ISSUE THREAD

**Issue by awsom82** [mattermost/mattermost-docker#369]
Can we have a major stable version? I mean, when `5.7.2` comes out, can it be also tagged as `5.7` and just `5`. To have a better lockup by major version in `docker-compose.yml`. So, it wouldn't be needed to edit a `docker-compose.yml` each time new minor versions come out.

Igor.

**Comment by pichouk** [comment]
I agree it could be a good idea, but I don't know if we can do this with the Docker Hub auto-build from the Github repository. @carlosasj Do you think it could be possible ?

**Comment by greatfireball** [comment]
Indeed it would be possible using a custom hook. I am building my docker images and tag it later as latest to ensure latest is always my latest build.

An example `post_push` hook is available in my [hooks repository](https://github.com/greatfireball/docker_hooks). If required, I can provide such a hook script to ensure tagging of new builds for different version numbers.

**Comment by awsom82** [comment]
@pichouk since docker 1.10 we can provide multiple image tags at build.
Example: `docker build -t mattermost-prod-app:5.7.2 -t mattermost-prod-app:5.7 -t mattermost-prod-app:5 -t mattermost-prod-app:latest .`

I'm not familiar with Docker Hub, but it seems it could be done via tags and regexp.
<img width="1079" alt="2019-02-17 20 07 54" src="https://user-images.githubusercontent.com/5804704/52914964-122d2880-32f0-11e9-95a7-788403c21300.png">

Hope this helps a little.
Igor.

**Comment by pichouk** [comment]
Could be great, I don't have the access to configure Github/Docker Hub hooks for the repository. Maybe @cpanato can ?

**Comment by greatfireball** [comment]
Just my two cents: In case that works, it will cause multiple build processes. Currently, I have some issues setting up my automated builds. I would go for the build hooks, which just adds additional tags to the last build.

**Comment by jaydeland** [comment]
I can take a look at this
